generate.py

```diff
@@ -143,6 +154,8 @@ def read_media_sub(vidfile, subfile, option):
             # CONFIG OPTION - contrast and brightness
             # enhance the image (increase contrast and brightness) for terminal display
             # TURN OFF (by commenting) IF YOU PREFER THE ORIGINAL COLOURS
+            if keyboard.is_pressed("s"):
```

Comment:
Hey @JeremiasFuentes, here it is checking for the "s" key is pressed or not. So have you implemented different keystrokes for different modes(subtitle and non-subtitle)?